meta-ibm/meta-witherspoon/recipes-kernel/linux/linux-aspeed/0001-Fix-psu-signal-and-vout-issue.patch

```diff
@@ -22,36 +29,63 @@ index 7cf4cc5..844531e 100644
  			PMBUS_HAVE_STATUS_VOUT | PMBUS_HAVE_STATUS_IOUT |
  			PMBUS_HAVE_STATUS_INPUT | PMBUS_HAVE_STATUS_TEMP |
  			PMBUS_HAVE_STATUS_FAN12,
+@@ -469,7 +468,7 @@ static struct pmbus_driver_info ibm_cffps_info[] = {
+ };
+ 
+ static struct pmbus_platform_data ibm_cffps_pdata = {
+-	.flags = PMBUS_SKIP_STATUS_CHECK,
++	.flags = PMBUS_SKIP_STATUS_CHECK | PMBUS_NO_CAPABILITY,
+ };
+ 
+ static int ibm_cffps_probe(struct i2c_client *client,
 diff --git a/drivers/hwmon/pmbus/pmbus_core.c b/drivers/hwmon/pmbus/pmbus_core.c
-index f7fc388..e9646c8 100644
+index f7fc388..418e070 100644
 --- a/drivers/hwmon/pmbus/pmbus_core.c
 +++ b/drivers/hwmon/pmbus/pmbus_core.c
-@@ -636,8 +636,10 @@ static long pmbus_reg2data_linear(struct pmbus_data *data,
+@@ -635,7 +635,7 @@ static long pmbus_reg2data_linear(struct pmbus_data *data,
+ 	s32 mantissa;
  	long val;
  
- 	if (sensor->class == PSC_VOLTAGE_OUT) {	/* LINEAR16 */
--		exponent = data->exponent[sensor->page];
--		mantissa = (u16) sensor->data;
-+		//exponent = data->exponent[sensor->page];
-+		//mantissa = (u16) sensor->data;
-+                exponent = ((s16)sensor->data) >> 11;
-+                mantissa = ((s16)((sensor->data & 0x7ff) << 5)) >> 5;
```

Comment:
Do you not need this change? It's no-longer present on the right-hand side of the diff.

Reply:
I changed the conditions so that mowgli's PSU will use LINEAR11.